app/models/airtable_user.rb

```diff
@@ -1,6 +1,8 @@
 Airrecord.api_key = ENV['AIRTABLE_API_KEY']
 
 class AirtableUser < Airrecord::Table
+  audited
```

Comment:
I dont think this will work since this isnt on our DB, it's on AirTable.

Reply:
Yeah I wasn't sure if that was necessary. I was wondering if audited might track the differencez between each sync action